sechub-schedule/src/main/java/com/mercedesbenz/sechub/domain/schedule/SchedulerRestController.java

```diff
@@ -94,7 +88,7 @@ public SchedulerResult createJob(@PathVariable("projectId") String projectId, @R
 	public void uploadSourceCode(
 	        @PathVariable("projectId") String projectId,
 	        @PathVariable("jobUUID") UUID jobUUID,
-	        @RequestParam(MULTIPART_FILE) MultipartFile file,
+	        @RequestPart(MULTIPART_FILE) MultipartFile file,
```

Comment:
In a Spring Boot application, when handling file uploads (especially with multipart/form-data), you should use @RequestPart instead of @RequestParam. Here's why:
- @RequestPart is designed to handle multi-part file data. It allows you to work with files as well as other non-file fields in a request body.
- @RequestParam, on the other hand, is typically used for query parameters, form data, or simple form fields in a request.

Both of the Annotations can be used, however, you can make your choice about them based on how they interpret arguments internally.

From the official Spring Docs:

_The main difference is that when the method argument is not a String, @RequestParam relies on type conversion via a registered Converter or PropertyEditor while @RequestPart relies on HttpMessageConverters taking into consideration the 'Content-Type' header of the request part. @RequestPara is likely to be used with name-value form fields while @RequestPart is likely to be used with parts containing more complex content (e.g. JSON, XML)._

I'd say that `RequestPart` is the better fit.